plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -91,6 +92,13 @@ public CheckpointWriter(TypeManager typeManager, CheckpointSchemaManager checkpo
 
     public void write(CheckpointEntries entries, TrinoOutputFile outputFile)
             throws IOException
+    {
+        write(entries, outputFile, ParquetWriterOptions.builder().build());
+    }
+
+    @VisibleForTesting
+    void write(CheckpointEntries entries, TrinoOutputFile outputFile, ParquetWriterOptions parquetWriterOptions)
```

Comment:
`ParquetWriterOptions` should probably be an argument to the constructor of `CheckpointWriter`. It seems safer to use the same options to write all files rather than allowing different config for every file.